data/json/mapgen_palettes/house_general_palette.json

```
@@ -0,0 +1,187 @@
+        { "item": "cleaning", "chance": 30, "repeat": [ 1, 2 ] },
+        { "item": "mechanics", "chance": 5, "repeat": [ 1, 2 ] }
+      ],
+      "r": [ { "item": "office_paper", "chance": 5 }, { "item": "office", "chance": 25 } ],
+      "t": { "item": "SUS_toilet", "chance": 10, "repeat": [ 1, 3 ] },
+      "v": [
+        { "item": "art", "chance": 5 },
+        { "item": "harddrugs", "chance": 10, "repeat": [ 1, 2 ] },
+        { "item": "maps", "chance": 2 },
+        { "item": "guns_pistol_common", "chance": 50, "ammo": 90, "magazine": 100 },
+        { "item": "gemstones", "chance": 30, "repeat": [ 1, 2 ] }
+      ],
+      "w": [
+        { "item": "shower", "chance": 20 },
+        { "item": "cleaning", "chance": 30, "repeat": [ 1, 2 ] },
+        { "item": "bed", "chance": 30, "repeat": [ 1, 2 ] }
+      ],
+      "x": { "item": "elecsto_persele", "chance": 10, "repeat": [ 1, 3 ] },
+      "B": { "item": "shower", "chance": 10, "repeat": [ 1, 2 ] },
+      "D": [
+        { "item": "SUS_wardrobe_mens", "chance": 50, "repeat": [ 1, 2 ] },
+        { "item": "SUS_wardrobe_womens", "chance": 50, "repeat": [ 1, 2 ] }
+      ],
+      "F": { "item": "SUS_fridge", "chance": 100 },
+      "I": [ { "item": "SUS_desks_bedroom_unisex", "chance": 5 }, { "item": "office", "chance": 25 } ],
+      "J": { "item": "kitchen_counters", "chance": 5 },
+      "L": { "item": "unisex_coat_rack", "chance": 30, "repeat": [ 1, 4 ] },
+      "N": [
+        { "item": "home_hw", "chance": 20, "repeat": [ 1, 2 ] },
+        { "item": "cleaning", "chance": 30, "repeat": [ 1, 2 ] },
+        { "item": "mechanics", "chance": 10, "repeat": [ 1, 2 ] }
+      ],
+      "O": { "item": "SUS_oven", "chance": 100 },
+      "P": { "item": "office_paper", "chance": 100 },
+      "Q": [
+        { "item": "shower", "chance": 20 },
+        { "item": "cleaning", "chance": 30, "repeat": [ 1, 2 ] },
+        { "item": "bed", "chance": 30, "repeat": [ 1, 2 ] }
+      ],
+      "R": { "item": "homebooks", "chance": 30, "repeat": [ 1, 4 ] },
+      "S": { "item": "SUS_bathroom_sink", "chance": 75 },
+      "U": [
+        { "item": "home_hw", "chance": 20, "repeat": [ 1, 2 ] },
+        { "item": "cleaning", "chance": 30, "repeat": [ 1, 2 ] },
+        { "item": "mechanics", "chance": 10, "repeat": [ 1, 2 ] }
+      ],
+      "Y": { "item": "trash", "chance": 30, "repeat": [ 1, 4 ] },
+      "1": [ { "item": "SUS_dishes", "chance": 100 }, { "item": "SUS_silverware", "chance": 100 } ],
+      "2": { "item": "SUS_cookware", "chance": 100 },
+      "3": [ { "item": "SUS_utensils", "chance": 50 }, { "item": "SUS_knife_drawer", "chance": 50 } ],
+      "4": { "item": "SUS_junk_drawer", "chance": 100 },
+      "5": { "item": "SUS_kitchen_sink", "chance": 100 },
+      "6": { "item": "SUS_pantry", "chance": 25, "repeat": [ 1, 2 ] },
+      "7": [ { "item": "SUS_breakfast_cupboard", "chance": 100 }, { "item": "SUS_coffee_cupboard", "chance": 100 } ],
```

```suggestion
      "7": [ { "item": "SUS_breakfast_cupboard", "chance": 60 }, { "item": "SUS_coffee_cupboard", "chance": 60 } ],
```
At some point we could think about making this and some of the others into a distribution that is guaranteed to spawn, and has either a breakfast cupboard, a coffee cupboard, or a small chance of both.